client/src/components/Workflow/Published/WorkflowPublished.vue

```diff
@@ -0,0 +1,222 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faBuilding, faDownload, faEdit, faPlay, faSpinner, faUser } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import axios, { type AxiosError } from "axios";
+import { computed, onUnmounted, ref, watch } from "vue";
+
+import { fromSimple } from "@/components/Workflow/Editor/modules/model";
+import { useDatatypesMapper } from "@/composables/datatypesMapper";
+import { usePanels } from "@/composables/usePanels";
+import { useUserStore } from "@/stores/userStore";
+import { useConnectionStore } from "@/stores/workflowConnectionStore";
+import { useWorkflowStateStore } from "@/stores/workflowEditorStateStore";
+import { useWorkflowStepStore } from "@/stores/workflowStepStore";
+import type { Workflow } from "@/stores/workflowStore";
+import { assertDefined } from "@/utils/assertions";
+import { withPrefix } from "@/utils/redirect";
+
+import WorkflowInformation from "./WorkflowInformation.vue";
+import ActivityBar from "@/components/ActivityBar/ActivityBar.vue";
+import Heading from "@/components/Common/Heading.vue";
+import FlexPanel from "@/components/Panels/FlexPanel.vue";
+import ToolBox from "@/components/Panels/ProviderAwareToolBox.vue";
+import WorkflowGraph from "@/components/Workflow/Editor/WorkflowGraph.vue";
+
+library.add(faSpinner, faUser, faBuilding, faPlay, faEdit, faDownload);
+
+const props = defineProps<{
+    id: string;
+}>();
+
+const workflowInfo = ref<
+    | {
+          name: string;
+          [key: string]: unknown;
+          license?: string;
+          tags?: string[];
+          update_time: string;
+      }
+    | undefined
+>();
+
+const workflow = ref<Workflow | null>(null);
+
+const loading = ref(true);
+const errored = ref(false);
+const errorMessage = ref("");
+
+const { datatypesMapper } = useDatatypesMapper();
+
+const connectionsStore = useConnectionStore();
+const stepStore = useWorkflowStepStore();
+const stateStore = useWorkflowStateStore();
```

Comment:
Do we need to scope these stores somehow, e.g. to the workflow id ? It's definitely not a blocker, but if we start using them throughout more parts of the app I guess we need to be careful ?

Reply:
Yes, I think scoping these to the workflow id is the way to go. I'd like to handle that in a separate PR however.